### ISSUE TITLE
no alt text defined for the owncloud logo.

### ISSUE THREAD

**Issue by davidleebooth** [guillaumev/owncloud_for_filelink#46]
In running mail testing on my emails, I received notice that the owncloud logo does not have an alt text attribute in the html. Though minor, this can have an effect on an emails spam score. would it be possible to plug in alt="Cloud Logo" or something simple to appease the email gods?

**Comment by guillaumev** [comment]
Closing this as won't fix since this is not something I have control over. The HTML is added automatically by thunderbird, so I think this more an issue that should be raised to the thunderbird filelink project.
